### PR TITLE
Add SM/USUM pokedex data

### PR DESCRIPTION
**Pull request by codedmonkey**
Data courtesy of Bulbapedia.

Late edit: I understand the data in this repository shouldn't come from other fansites, but since I wanted the data anyway I opted to try to give back to this awesome dataset. I checked the edit history of the Pokedex page on Bulbapedia and it seems they never had to make a change, so I suspect they have a good source although I must admit I didn't check them all individually.

### CODE REVIEW

**Comment by magical** [comment]
LGTM, from a cursory glance.

You're correct that we usually prefer not to scrape data from other sites, but in this case, pokedex numbers are so visible that the risk of them being inaccurate is probably pretty small.

File: pokedex/data/csv/pokemon_dex_numbers.csv [hunk 1]
```diff
@@ -2219,41 +3161,73 @@ species_id,pokedex_id,pokedex_number
 422,1,422
 422,5,60
 422,6,60
+422,21,341
+422,25,88
+422E,16,260
+422E,20,75
```
Comment (magical) [inline pokedex/data/csv/pokemon_dex_numbers.csv]: These two rows are invalid. `422E` should be `422`.

File: pokedex/data/csv/pokemon_dex_numbers.csv [hunk 2]
```diff
@@ -2219,41 +3161,73 @@ species_id,pokedex_id,pokedex_number
...
 423,1,423
 423,5,61
 423,6,61
+423,21,342
+423,25,89
+423E,16,261
+423E,20,76
```
Comment (magical) [inline pokedex/data/csv/pokemon_dex_numbers.csv]: Same here.

File: pokedex/data/csv/pokemon_dex_numbers.csv (hunk 1 above)
Comment (codedmonkey) [reply pokedex/data/csv/pokemon_dex_numbers.csv]: Thanks, somehow missed those.

**Comment by magical** [comment]
Thanks!

**Pull request merged**